protostar/commands/test/test_command.py

```diff
@@ -245,7 +254,24 @@ async def test(
                 "Only one test case can be profiled at the time. Please specify path to a single test case."
             )
 
-        self._log_test_collector_result(test_collector_result)
+        if structured_format:
+            test_cases = 0
+            for test_suite in test_collector_result.test_suites:
+                test_cases += len(test_suite.test_cases)
```

Comment:
Please add `_count` prefix, test_cases suggests list with `TestCase` type.

[[Nitpick]] Can we also move out counting from this function? It clutters the code. I would add `test_cases_count` property to TestSuite and then write it like this.

`test_cases_count = sum(suite.test_cases_count for suite in test_collector_result.test_suites)`

Reply:
you mean suffix, right? :D 

What you proposed doesn't really change anything. If I change my code to a one-liner like that, we will have:
```
test_cases_count = [sum(len(test_suite.test_cases) for test_suite in test_collector_result.test_suites)]
```
So we'd end up just adding the field `test_cases_count ` to `TestSuite` which would basically be `len(test_cases)`, right?

Reply:
Also, lint keeps complaining: "consider using a generator" for this (I think I probably mentioned this in this PR), I don't think a simple for loop here is such a big deal overall.

Reply:
naming is resolved in [format + names + remove redundant function](https://github.com/software-mansion/protostar/pull/1254/commits/f62b551533db9762d7ce549c5fa2823cc8965a14)

Please, let me know if you're okay with leaving the for loop

Reply:
>Please, let me know if you're okay with leaving the for loop
Ofc I am, It is marked as nitpick 😉

Reply:
FYI
```
 test_cases_count = sum(len(test_suite.test_cases) for test_suite in test_collector_result.test_suites)
```
You do not need square brackets, then lint will not complain and sum is executed lazily. I dont heave heavy preference between your and this solution

Reply:
Right, this works, thanks!

[use one-liner](https://github.com/software-mansion/protostar/pull/1254/commits/2c2ed597f95ecca8013b75fc9366028b065a49df)